e2e-tests/portal/cypress/e2e/journeys/maker-facilities/AIN-facility-cover-start-date-validation-past-3-months.spec.js

```diff
@@ -19,9 +20,9 @@ context('Issue facilities beyond 3 months of submission - errors', () => {
   const month = date.getMonth() + 1;
   const year = date.getFullYear();
 
-  const { submissionDate } = AINDeal.details;
-  const submissionDateFormatted = moment(new Date(submissionDate)).format('Do MMMM yyyy');
-  const submissionDatePlus3Months = moment(submissionDate).add(3, 'month').format('Do MMMM yyyy');
+  const submissionDate = new Date(AINDeal.details.submissionDate);
+  const submissionDateFormatted = format(submissionDate, 'do MMMM yyyy');
```

Comment:
Do you think it will help to have these common formats in constants to move closer to one standardised date.